src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -218,7 +218,13 @@ public static ItemStack getItemStack(JsonObject json, JsonContext context)
             // Lets hope this works? Needs test
             try
             {
-                NBTTagCompound nbt = JsonToNBT.getTagFromJson(GSON.toJson(json.get("nbt")));
+                JsonElement element = json.get("nbt");
+                NBTTagCompound nbt;
+                if(element.isJsonObject())
```

Comment:
I didnt use { } because the other single-line if's in the file dont use them either
* https://github.com/Xalcon/MinecraftForge/blob/427cae4430346b4627515839ba18b5f17d26dbed/src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java#L223-L226
* https://github.com/Xalcon/MinecraftForge/blob/427cae4430346b4627515839ba18b5f17d26dbed/src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java#L282-L285
* https://github.com/Xalcon/MinecraftForge/blob/427cae4430346b4627515839ba18b5f17d26dbed/src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java#L613-L616